types/session_price.yaml

```
@@ -0,0 +1,73 @@
+description: Types to provide the session price
+types:
+  SessionCostChunk:
+    description: >-
+      A chunk of the session cost. The total cost of the session is the sum of
+      all chunks.
+    type: object
+    additionalProperties: false
+    properties:
+      timestamp_from:
+        description: >-
+          Absolute timestamp for the start of this chunk in RFC3339 UTC format
+        type: string
+        format: date-time
+      timestamp_to:
+        description: >-
+          Absolute timestamp for the end of this chunk in RFC3339 UTC format
+        type: string
+        format: date-time
+      cost_energy:
+        description: >-
+          Cost of the energy consumed during this chunk in the currency
+          specified in the session price
+        type: number
+      cost_time:
+        description: >-
+          Cost of the time spent during this chunk in the currency specified
+          in the session price
+        type: number
+  SessionStatus:
+    description: >-
+      Session status enum. Session can be running or finished. Costs of the
+      running session are not final and can change.
+    type: string
+    enum:
+      - running
+      - finished
```

```suggestion
      - Running
      - Finished
```
Nit: first letter capitalized in enum values

---

Done